### PR TITLE
Update ruby docs

### PR DESCRIPTION
**Pull request by st0012**

### CODE REVIEW

File: src/platforms/ruby/common/configuration/options.mdx [hunk 1]
```diff
@@ -140,6 +140,14 @@ config.logger = Sentry::Logger.new(STDOUT)
 
 Sentry respects logger levels.
 
+`max_breadcrumbs`
+
+: The maximum number of breadcrumbs the SDK would hold. Default is `100`.
```
Comment (PeloWriter) [inline src/platforms/ruby/common/configuration/options.mdx:145]:
```suggestion
: This variable controls the total amount of breadcrumbs that should be captured. The default is `100`.
```

**Review by PeloWriter — APPROVED** [review]
minor change to breadcrumb wording. Thanks, @st0012

**Comment by st0012** [comment]
@bruno-garcia can you also give it a quick scan and help me merge this PR? thanks 😄

File: src/includes/performance/traces-sampler-as-sampler/ruby.mdx [hunk 2]
```diff
@@ -4,14 +4,20 @@ Sentry.init do |config|
...
-    transaction = sampling_context[:transaction_context]
+    transaction_context = sampling_context[:transaction_context]
 
-    # if the transaction is important, set a higher rate
-    if transaction[:name].match?("orders")
-      0.5
-    # otherwise, give it a lower rate
-    else
-      0.1
+    # with transaction_context, you can sample transactions more sophisticatedly
```
Comment (st0012) [inline src/includes/performance/traces-sampler-as-sampler/ruby.mdx:9]: @PeloWriter could you help me refine this example as well? thanks 🙏

File: src/includes/performance/traces-sampler-as-sampler/ruby.mdx [hunk 3]
```diff
@@ -4,14 +4,20 @@ Sentry.init do |config|
...
+    # for example, you can give different rates based on the operation
```
Comment (PeloWriter) [inline src/includes/performance/traces-sampler-as-sampler/ruby.mdx:10]:
```suggestion
    # for example, you can provide different sample rates based on the operation
```

File: src/includes/performance/traces-sampler-as-sampler/ruby.mdx (hunk 2 above)
Comment (PeloWriter) [inline src/includes/performance/traces-sampler-as-sampler/ruby.mdx:9]:
```suggestion
    # transaction_context helps you sample transactions with more sophistication
```

Comment (PeloWriter) [reply src/includes/performance/traces-sampler-as-sampler/ruby.mdx:9]: Sure :)

**Review by PeloWriter — COMMENTED** [review]
minor wording changes. I'll leave technical review to the team :)

**Review by lobsterkatie — APPROVED** [review]
LGTM!

**Comment by PeloWriter** [comment]
Thanks, @st0012 !

**Comment by st0012** [comment]
@PeloWriter thanks for merging this, but I think a change is lost when merging with master 😅 I'll open another PR for that

**Pull request merged**